lib/hanami/cli/commands.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require "hanami/cli"
+
+module Hanami
+  # Hanami CLI
+  #
+  # @since 1.1.0
+  class CLI
+    # Register a command to expand Hanami CLI
+    #
+    # @param name [String] the command name
+    # @param command [NilClass,Hanami::CLI::Command,Hanami::CLI::Commands::Command]
+    #   the optional command
+    # @param aliases [Array<String>] an optional list of aliases
+    #
+    # @since 1.1.0
+    #
+    # @example Third party gem
+    #   require "hanami/cli/commands"
+    #
+    #   module Hanami
+    #     module Webpack
+    #       module CLI
+    #         module Commands
+    #           class Generate < Hanami::CLI::Command
+    #             desc "Generate Webpack config"
```

Comment:
```suggestion
    #             desc "Generate Webpack configuration"
```

I assume this is the string that should match the one in line 48, right?

Reply:
@kaikuchn That's correct, good catch 👍